cli/cmd/errors.go

```diff
@@ -77,9 +80,28 @@ func ErrorCortexYAMLNotFound() error {
 }
 
 func ErrorDockerDaemon() error {
```

Comment:
This name sounds a a bit vague, can you improve the name of this error function and the kind.